README.md

```diff
@@ -1,9 +1,11 @@
-![UI/UX Designer Roadmap](http://i.imgur.com/clLNiOD.png)
+![UI/UX Designer Roadmap 2019](https://i.imgur.com/DYkQ9MB.png)
 
-> Roadmap to becoming a UI/UX Designer in 2017
+> Roadmap to becoming an UI/UX Designer in 2019
 
 Below, you'll find a roadmap on how to become a UI/UX designer. It lists all common tools of the trade in order to become a digital creative. It is inspired by [this roadmap on how to become a web developer](https://github.com/kamranahmedse/developer-roadmap).
 
+this is an updated version of the [Roadmap to becoming an UI/UX Designer 2017](https://github.com/togiberlin/designer-roadmap).
```

Comment:
Please remove that line, it is unnecessary.

Reply:
sure!